polaris-tokens/src/token-groups/color.ts

```diff
@@ -430,106 +430,132 @@ export const color: {
   },
   'color-border-input-hover': {
     value: colors.gray[800],
+    valueExperimental: colorsExperimental.gray[13],
     description: '',
   },
   'color-border-inverse': {
     value: colors.gray[800],
+    valueExperimental: colorsExperimental.gray[13],
     description: '',
   },
   'color-border-strong-hover': {
     value: colors.gray[700],
+    valueExperimental: colorsExperimental.gray[10],
     description: '',
   },
   'color-border-input': {
     value: colors.gray[600],
+    valueExperimental: colorsExperimental.gray[12],
     description: '',
   },
   'color-border-hover': {
     value: colors.gray[600],
+    valueExperimental: colorsExperimental.gray[10],
     description: '',
   },
   'color-border-strong': {
     value: colors.gray[600],
+    valueExperimental: colorsExperimental.gray[9],
     description: '',
   },
   'color-border': {
     value: colors.gray[500],
+    valueExperimental: colorsExperimental.gray[8],
     description: '',
   },
   'color-border-disabled': {
     value: colors.gray[400],
+    valueExperimental: colorsExperimental.gray[7],
     description: '',
   },
   'color-border-subdued': {
     value: colors.gray[400],
+    valueExperimental: colorsExperimental.gray[7],
     description: '',
   },
   'color-border-interactive-disabled': {
     value: colors.gray[400],
+    valueExperimental: colorsExperimental.gray[7],
     description: '',
   },
   'color-border-primary': {
     value: colors.green[700],
+    valueExperimental: colorsExperimental.green[8],
```

Comment:
Looks like `border-primary` maps to `GR08` in the [Figma](https://www.figma.com/file/hL9XRyHGqU8wUtLdOgs0vI/Polaris-Uplift-Library?type=design&node-id=9011%3A154956&t=P03VetQBVKbERO5B-1)